### DISCUSSION TITLE
The ultimate farm [Closed]

### DISCUSSION THREAD

**Discussion by dandev10** [Nikegamerjjjj/plantclicker#2] [Ideas]
Make a ultimate farm. To unlock u e end 5000 plants. How many  it gives? 1000. Hmm think about it

**Comment by nikeedev** [comment]
**Hello @THEspager! I thank you for the idea, I will take care of it, and see if I can do that. Otherwise, have a good day further!**

***Nikita***

**Comment by nikeedev** [comment]
> Make a ultimate farm. To unlock u e end 5000 plants. How many it gives? 1000. Hmm think about it

**Is it okay if price will be 500 plants and will give 100 plants each second?**

***Nikita***

**Reply by dandev10** [reply]
> > Make a ultimate farm. To unlock u e end 5000 plants. How many it gives? 1000. Hmm think about it
> 
> **Is it okay if price will be 500 plants and will give 100 plants each second?**
> 
> _**Nikita**_

Man I want 550 plants to buy okay also it’s gonna give 250 plants also it’s 1:00 am in New York

**Reply by nikeedev** [reply]
*Okay, that’s great and also great price! Good to know that New York has 1.00 am at time.*

***Nikita***
